sda/internal/database/db_functions_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"regexp"
 
 	"github.com/google/uuid"
+	"github.com/lib/pq"
```

Comment:
```suggestion
```
This is already imported in `database.go`

Reply:
But as `_`, so I can't use the functions from the lib without importing it again..?

Reply:
It will only needs to be imported in `db_functions.go`